modules/ROOT/pages/split-aggregate-scope.adoc

```diff
@@ -0,0 +1,108 @@
+
+The Split Aggregate scope can be configured through the following fields described in this table:
+
+[%header,cols="1,3"]
+|===
+|Child element |Description
+|Collection (`collection`) | Specifies the expression that defines the collection of parts to be processed in parallel. By default, it will use the incoming payload.
+|==
+
+[%header,cols="1,3"]
+|===
+|Attribute |Description
+|Timeout (`timeout`) | Specifies the timeout for each parallel route. By default there is no timeout.
+|Max Concurrency (`maxConcurrency`) | Specifies the maximum level of parallelism that will be used by this router. By default, all routes will run in parallel.
+|Target Variable (`target`) | Specifies the variable where to save the processed payload. By default the output will be saved in the flow's payload. 
+|Target Value (`targetValue`) | Specifies an expression to evaluate against the operation's output value. The outcome of this expression will be stored in the target variable. By default, this will be the same as the operation's output value.
+|===
+
+This XML example adds to every element in the collection the string `"-result"`: 
+
+----
+<split-aggregate>
+	<collection>
+		#[['apple', 'banana', 'orange']]
+	</collection>
+	<set-payload value="#[payload ++ '-result']"/>
+</split-aggregate>
+
+----
+
+Every route starts with the same initial variable values. If a variables is modified within an specific route, this will not affect other routes. If a variable is added or modified in one route then, after aggregation, the value will be the defined by that route. If a variable is added or modified by more than one route then the  value will be a list of all the values defined for that variable within all the routes. 
+For example:
+
+----
+<set-variable variableName="var1" value="var1"/>
+<set-variable variableName="var2" value="var2"/>
+<split-aggregate>
+	<collection>
+		#[['apple', 'banana', 'orange']]
+	</collection>
+	<choice>
+		<when expression="#[payload == 'apple']">
+			<set-variable variableName="var2" value="newValue"/>
+			<set-variable variableName="var3" value="val1"/>
+
+		</when>
+		<when expression="#[payload == 'banana']">
+			<set-variable variableName="var3" value="val2"/>
+		</when>
+		<otherwise>
+			<set-variable variableName="var3" value="val3"/>
+			<set-variable variableName="var4" value="val4"/>
+		</otherwise>
+	</choice>
+</split-aggregate>
+<!-- After aggregation, the variables will be: 
+{var1: "var1", var2: "newValue", var3: ["val1, val2, val3"], var4: "val4"} -->
+----
+
+== Error Handling
+
+When an error is thrown in one route, since every route is being processed in parallel, Split Aggregate will continue t process each route and then invoke the xref:intro-error-handlers.adoc[Error Handler], with the aggregation of the results and errors of all the routes. For example:
```

Comment:
```suggestion
When an error is thrown in one route, since every route is being processed in parallel, Split Aggregate will continue to process each route and then invoke the xref:intro-error-handlers.adoc[Error Handler], with the aggregation of the results and errors of all the routes. For example:
```